src/bots/makerBidAskTwapCrank.ts

```diff
@@ -381,6 +413,17 @@ export class MakerBidAskTwapCrank implements Bot {
 					}),
 				];
 
+				if (
+					this.pythPriceSubscriber &&
+					!isVariant(
+						this.driftClient.getPerpMarketAccount(mi)!.amm.oracleSource,
+						'prelaunch'
```

Comment:
should you check that oracle source is one of pyth pull types?

Reply:
we want to first test cranking all of the oracles that arent prelaunch because there is a pyth id for nearly all of them.

Reply:
what if the feed id isnt set a la dsol

Reply:
will just return empty ixs to prepend -- shouldnt fall over.